corehq/apps/app_manager/migrations/0006_multi_master_linked_apps.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import absolute_import, unicode_literals
```

Comment:
FYI, Simon has some feedback on this here: https://github.com/dimagi/commcare-hq/pull/25107#pullrequestreview-288015289

Reply:
updated